docs/03-AcceptanceTests.md

```diff
@@ -310,13 +319,65 @@ $I->seeInCurrentUrl('user/1');
 $user_id = $I->grabFromCurrentUrl('~$/user/(\d+)/~');
 ```
 
-## Selenium WebDriver
+## WebDriver: Selenium Server or PhantomJS
 
 A nice feature of Codeception is that most scenarios are similar, no matter of how they are executed.
 `PhpBrowser` was emulating browser requests but how to execute such test in a real browser like Chrome or Firefox? 
 Selenium WebDriver can drive them so in our acceptance tests we can automate scenarios we used to test manually.
 In such tests we should concentrate more on **testing the UI** than on testing functionality.
 
+"[Selenium WebDriver](http://www.seleniumhq.org/projects/webdriver/)" is the name of an API (specified by the Selenium project)
+to drive browsers automatically. Codeception supports two implementations of this API: Selenium Standalone Server and PhantomJS
```

Comment:
OK, thanks! 2 things to PhantomJS - users' perspective:

* It's installed in 2 minutes. And the transition to Chrome is probably straightforward too - since no changes in the test files (Codeception syntax) are necessary.

* Noteworthy is the fact that is just doesn't make sense anymore to "debug" test code to get it running on Phantom. So the message is: If Phantom works for you, keep it for the moment. If it doesn't work, don't try to get it to work, but rather go with Selenium for now.

I'll include such a note. But please merge all pull requests first - I'm again losing track of what the status is and what needs to be done....